runner/Xpress/Project.toml

```diff
@@ -0,0 +1,12 @@
+[deps]
+HolisticElectricityModel = "02f1d289-301b-46a9-893d-51a2c7ce54b6"
+HolisticElectricityModelData = "cbe3faf1-a446-4ba2-bb0b-4327c65cbbd3"
+Ipopt = "b6b21f68-93f8-5de0-b562-5493be1d77c9"
+JuMP = "4076af6c-e467-56ae-b986-b466b2749572"
+Revise = "295af30f-e4ad-537b-8983-00126c2a3abe"
+Xpress = "9e70acf3-d6c9-5be6-b5bd-4e2c73e3e054"
+YAML = "ddb6d928-2868-570f-bddf-ab3f9cf99eb6"
+
+
+[compat]
+Xpress= "~0.16"
```

Comment:
Pinned the Xpress version here since 0.17 was throwing errors with JuMP (maybe an apple silicon problem?)